programming-languages/README.md

```diff
@@ -24,4 +24,29 @@
 ### Tools
 
 * [Compiler Explorer](https://godbolt.org/): Online C++ compiler. It lets you compile small code snippets and look at the generated assembly code. It supports different compilers across various architectures ranging from x86 to Arduinos.
-* [JetBrains CLion](https://www.jetbrains.com/clion/): One of the best C++ IDEs. CLion works on CMake making it easy to use with roscpp. 
\ No newline at end of file
+* [JetBrains CLion](https://www.jetbrains.com/clion/): One of the best C++ IDEs. CLion works on CMake making it easy to use with roscpp. 
+
+
+## Python
+A high level, interpreted, dead-simple, reads-like-english programming language used for a multitude of tasks, from Machine Learning/Data Analysis to deploying Full-Stack Websites.
+
+The [official website](https://www.python.org/) offers further resources, including talks, podcasts, conferences and the like.
+
+### Tutorials
+
```

Comment:
* [Corey Schafer Python Series](https://youtu.be/YYXdXT2l-Gg): One of the most elegant courses on python. Covers python from basic to intermediate.  Covers topics like generators, threading on top of all the basic concepts. It is highly recommended that you code along with the videos.